javav2/example_code/inspector/src/main/java/com/java/inspector/InspectorScenario.java

```diff
@@ -0,0 +1,196 @@
+
+        // Step 5: Create a findings filter (example)
+        System.out.println(DASHES);
+        System.out.println("Step 5: Creating a findings filter...");
+        try {
+            inspectorActions.createFilter(
+                    inspectorClient,
+                    "Suppress low severity findings for demo purposes"
+            );
+
+            System.out.println("Created example filter");
+
+        } catch (Exception e) {
+            System.err.println(" Could not create example filter: " + e.getMessage());
+        }
+        System.out.println();
+        waitForInputToContinue(scanner);
+
+        // Step 6: List existing filters
+        System.out.println(DASHES);
+        System.out.println("Step 6: Listing existing filters...");
+
+        try {
+            inspectorActions.listFilters(inspectorClient, 10);
+
+        } catch (Exception e) {
+            System.err.println(" Could not create example filter: " + e.getMessage());
+        }
+        System.out.println();
+        waitForInputToContinue(scanner);
+
+        // Step 7: Show usage totals
+        System.out.println(DASHES);
+        System.out.println("Step 7: Checking usage and costs...");
+
+        try {
+            inspectorActions.listUsageTotals(inspectorClient, null, 10);
+        } catch (Exception e) {
+            System.err.println(" Could not create example filter: " + e.getMessage());
+        }
+
+        System.out.println();
+        waitForInputToContinue(scanner);
+
+        // Step 8: Coverage statistics
+        System.out.println(DASHES);
+        System.out.println("Step 8: Getting coverage statistics...");
+
+        try{
+            inspectorActions.listCoverageStatistics(inspectorClient);
+        } catch (Exception e) {
+            System.err.println(" Could not create example filter: " + e.getMessage());
+        }
+        System.out.println();
+
+        System.out.println(" Inspector Basics scenario completed successfully!");
+        System.out.println();
+        System.out.println(" What you learned:");
+        System.out.println("   ✓ How to check Inspector account status and configuration");
+        System.out.println("   ✓ How to enable Inspector for different resource types");
+        System.out.println("   ✓ How to list and analyze security findings");
+        System.out.println("   ✓ How to monitor scan coverage across your resources");
+        System.out.println("   ✓ How to create filters to suppress findings");
+        System.out.println("   ✓ How to track usage and costs");
+        System.out.println();
+        System.out.println("🔗 Next steps:");
+        System.out.println("   • Set up EventBridge rules to respond to findings");
+        System.out.println("   • Create custom filters for your environment");
+        System.out.println("   • Generate detailed findings reports");
+        System.out.println("   • Integrate with AWS Security Hub");
+        waitForInputToContinue(scanner);
+
```

Comment:
Needs a cleanup method that removes the example filters and gives the option to disable inspector.